core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation.directive.html

```diff
@@ -26,9 +26,115 @@
     text-align: center;
     width: 46px;
 }
+.oppia-dropdown-toggle {
+  color: #ffffff !important;
```

Comment:
Can you try to not have `!important`? We really want to avoid this.

Reply:
@kevinlee12 I have made the necessary changes and I have removed !important.